utils/systemd-redis_server.service

```diff
@@ -1,4 +1,4 @@
-# example systemd service unit file for redis-server
+# example systemd service unit file for placeholderkv-server
```

Comment:
This file is covered here: https://github.com/placeholderkv/placeholderkv/pull/29/files